esmvalcore/_data_finder.py

```diff
@@ -124,16 +124,50 @@ def get_start_end_date(filename):
     return start_date, end_date
 
 
+def dates_to_timerange(start_date, end_date):
```

Comment:
I mean you can use this one if you think it's better, but I would choose between having one function or the other everywhere. Instead of having duplicated functions.

Reply:
Right, good point, I wasn't aware of that function. Will change that :+1: